app/lib/constants/colors.ts

```diff
@@ -12,6 +12,15 @@ export const SWITCH_TRACK_COLOR = {
 	true: '#2de0a5'
 };
 
+const pushTroubleshooting = {
+	pushConsumptionOnSuccess: '#148660',
```

Comment:
use figma colors name

Reply:
I understand, but there are already `successColor` and `dangerColor` in this file, if I just added `onSuccess` or `onDanger` it would be weird. The `dangerColor` is used in other places of the app and the difference between the names will be minimal, but there is a difference between the colors. WDYT?

Reply:
It's not strange, the right thing is to correct the mistake of the past and not make the same mistake again because it was wrong in the past.

Reply:
Even a good thing would be to check the colors used in figma, Jose is using some deprecated colors, you could ask him to change and use the colors that are already in the new design system.